CONTRIBUTING.md

```diff
@@ -15,6 +15,10 @@ This project and everyone participating in it is governed by the [CloudQuery Cod
 
 ## What To Know Before Getting Started
 
+### Open-core vs Open-source
+
+The CloudQuery framework, SDK and CLI are open source while plugins available under `plugins` are **open core**, hence not all contributions to plugins directory will be accepted if they are part of the commercial plugin offering - please file an issue before opening a PR. 
```

Comment:
maybe make `file an issue` a link to our issues?

Reply:
Done in commits/1e58e200365588e3103e337c63fa0f7c61bc65e1